### ISSUE TITLE
normalization is done out-of-order

### ISSUE THREAD

**Issue by dan-rl1976** [learn-co-curriculum/dsc-visualizing-confusion-matrices-lab#1]
## Link to Canvas

https://learning.flatironschool.com/courses/4341/assignments/141604?module_item_id=296770

## Issue Subtype

- [x ] Master branch code
- [ ] Solution branch code
- [ ] Code tests
- [ ] Layout/rendering issue
- [ ] Instructions unclear
- [ ] Other (explain below)

### Source

```
import pandas as pd
import matplotlib.pyplot as plt
from sklearn.linear_model import LogisticRegression
from sklearn.model_selection import train_test_split

# Load the data
df = pd.read_csv('heart.csv')

# Define appropriate X and y
X = df[df.columns[:-1]]
y = df.target

# Normalize the data
for col in df.columns:
    df[col] = (df[col] - min(df[col]))/ (max(df[col]) - min(df[col]))

# Split the data into train and test sets 
X_train, X_test, y_train, y_test = train_test_split(X, y, random_state=0)

# Fit a model
logreg = LogisticRegression(fit_intercept=False, C=1e12, solver='liblinear')
model_log = logreg.fit(X_train, y_train)

# Preview model params
print(model_log) 

# Predict
y_hat_test = logreg.predict(X_test)

print("")
# Data preview
df.head()
```

### Concern

The data we are using to fit our model has not been normalized.

## What OS Are You Using?

- [ x] OS X
- [ ] Windows
- [ ] WSL
- [ ] Linux
- [ ] IllumiDesk

**Comment by Cheffrey2000** [comment]
Issue has been resolved by moving the regularization to a position after the train-test-split.  Other cells were updated to reflect the output due to the change in code.
